src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator, Drawable {
```

Comment:
```suggestion
public class Circle extends Figure {
```
let's implement interfaces only in Figure class
child classes will have these methods automatically